AutoRest/Generators/CSharp/CSharp/Templates/ParameterGroupTemplate.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@using Microsoft.Rest.Generator
+@using Microsoft.Rest.Generator.CSharp
+@using Microsoft.Rest.Generator.CSharp.Templates
+@using Microsoft.Rest.Generator.Utilities
+@inherits Microsoft.Rest.Generator.Template<Microsoft.Rest.Generator.CSharp.ParameterGroupTemplateModel>
+@Header("// ")
+@EmptyLine
+namespace @Settings.Namespace
+{
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Rest;
```

Comment:
@yugangw-msft Good catch.  I had copied this from elsewhere -- they in fact aren't all required.  I have removed all of them except for `using System`.